src/owncloud/setup_owncloud

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+############################################
+# This script is called by /bin/envvars
+############################################
+#
+. $SNAP/setup/filesystem.sh
```

Comment:
Ideally, I'd like to be able to place this is a "snap setup script" so that it doesn't need to be added to every daemon.